tests/bleTestsScripts/testClass.py

```diff
@@ -438,6 +442,27 @@ def advertisement(testDevice, DUT_UUID=None):
 
         return True
 
+    @staticmethod
+    def check_manufacture_data(testDevice, times, DUT_UUID=None):
+        manufacture_data = bleAdapter.getPropertie(testDevice, "ManufacturerData")
+        if( manufacture_data == None ):
+            print("No Manufacture Data")
+            sys.stdout.flush()
+            if times == (runTest.MANUFACTURE_CHECK_CASES_NUMBER-1) :
```

Comment:
the usage of this "times" variable is hard to understand & maintain. Should we come up with a better way?